src/johnsession.cpp

```diff
@@ -173,7 +175,7 @@ bool JohnSession::save()
 
     // Advanced options
     if (isForkEnabled()) {
-        m_settings->setValue("nbForkProcess", m_nbProcess);
+        m_settings->setValue("nbForkProcess", m_fork);
```

Comment:
Amendment. Rename to m_forkProcesses and always update the settings key names to be consistent.